aspects/electrical/Converter/GunnsElectConverterInput.cpp

```diff
@@ -307,7 +307,7 @@ void GunnsElectConverterInput::restartModel()
 ///           it can be re-checked this pass.  Update sensors with the timestep to advance their
 ///           drift malfunction.  Then call minorStep for the main update functions.
 ////////////////////////////////////////////////////////////////////////////////////////////////////
-void GunnsElectConverterInput::step(const double dt)
+void GunnsElectConverterInput::step(const double)
```

Comment:
By convention in the rest of the repo we use the ```dt __attribute__((unused))``` syntax for unused vars, and comment that it's unused in the 'param' doxygen field above.  See the example of dt in the minorStep() function.

Reply:
Reverted.